### PR TITLE
O Bilynska IO

### PR DESCRIPTION
**Pull request by Bilunska**

### CODE REVIEW

File: src/main/java/com/company/io/Creator.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        Map <String, CellStyle> styles = new HashMap < > ();
+
+        CellStyle style = wb.createCellStyle();
+
+        style.setBorderBottom(BorderStyle.MEDIUM);
+        style.setBottomBorderColor(IndexedColors.BLACK.getIndex());
+        style.setBorderRight(BorderStyle.MEDIUM);
+        style.setRightBorderColor(IndexedColors.BLUE.getIndex());
+        style.setFillForegroundColor(IndexedColors.LAVENDER.getIndex());
+        style.setFillPattern(FillPatternType.SOLID_FOREGROUND);
```
Comment (AqaCoreSchool) [inline src/main/java/com/company/io/Creator.java:34]: I imagine this is beautiful but to exhausted to  compile can you drop an printscreen in telegram chat?)

File: src/main/java/com/company/io/Main.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            wb = new HSSFWorkbook();
+        } else wb = new XSSFWorkbook();
+
+        Map <String, CellStyle> styles = createStyles(wb);
+        Sheet sheet = wb.createSheet("Converter sheet");
+
+        wb.setPrintArea(0, "$A$1:$C$2");
+        sheet.setZoom(140);
+        sheet.autoSizeColumn(0);
+        for (int i = 0; i <= 10; i++) {
```
Comment (AqaCoreSchool) [inline src/main/java/com/company/io/Main.java]: shouldn't hardcode 10 here, what if theres 200 files? what if you dont know how many values to expect?

File: src/main/java/com/company/io/Creator.java (hunk 1 above)
Comment (MychkaO) [reply src/main/java/com/company/io/Creator.java:34]: Done)

File: src/main/java/com/company/io/Main.java (hunk 2 above)
Comment (MychkaO) [reply src/main/java/com/company/io/Main.java]: Fixed

**Pull request merged**